societe-archeologique-de-bordeaux.csl

```diff
@@ -0,0 +1,290 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" demote-non-dropping-particle="never" default-locale="fr-FR" page-range-format="expanded">
+  <info>
+    <title>Société Archéologique de Bordeaux</title>
+	    <title-short>SAB</title-short>
+    <id>http://www.zotero.org/styles/societe-archeologique-de-bordeaux</id>
+    <link href="http://www.zotero.org/styles/societe-archeologique-de-bordeaux" rel="self"/>
+    <link href="http://www.societe-archeologique-bordeaux.fr/publications/recommandations-aux-auteurs.html" rel="documentation"/>
+    <author>
+      <name>Hélène Réveillas</name>
+      <email>hreveillas@bordeaux-metropole.fr</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="history"/>
+    <summary>Style reprenant les normes diffusées par la SAB</summary>
+    <updated>2016-11-09T08:31:00+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="editor">
+        <single>(dir.) </single>
+        <multiple>(dir.)</multiple>
+      </term>
```

Comment:
No it's not correct, thank you very much.
I'm not sure I understand your remark. As "dir." or "éd." will always have to be with parentheses, isn't it better to put them in the term name ?
Thank you again !